src/mercury_engine_data_structures/formats/collision.py

```diff
@@ -0,0 +1,66 @@
+import construct
+from construct import Array, Flag, Hex, IfThenElse, Int16ul, Rebuild, Struct
+
+from mercury_engine_data_structures import game_check
+from mercury_engine_data_structures.common_types import CVector2D, Float, UInt, make_vector
+from mercury_engine_data_structures.construct_extensions.misc import OptionalValue
+from mercury_engine_data_structures.game_check import Game
+
+CollisionPoint = Struct(
+    x=Float,
+    y=Float,
+    material_attribute=UInt,
+)
+CollisionPolySR = Struct(
+    num_points=Rebuild(UInt, construct.len_(construct.this.points)),
+    unk4=Hex(construct.Byte),
+    unk5=Hex(UInt),
```

Comment:
what are the odds these two unknowns are the same as the unknown and the loop flag in dread?

Reply:
What is the meaning of the loop flag?
I have seen that the shapes end up "open" and kinda connects the last point with the first point in the list automatically...is this considered a "loop"?